test/e2e/storage/volume_provisioning.go

```diff
@@ -421,8 +421,8 @@ var _ = SIGDescribe("Dynamic Provisioning", func() {
 			gceCloud, err := framework.GetGCECloud()
 			Expect(err).NotTo(HaveOccurred())
 
-			// Get all k8s managed zones
-			managedZones, err = gceCloud.GetAllZones()
+			// Get all k8s managed zones (in this case it is just the single zone)
```

Comment:
I don't think that's actually true. If you will be running this test for multizone clusters, these actually can be more than one zone.